go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -555,20 +571,17 @@ go.app = (function () {
       if (answers.state_tracing) {
         if (risk === "low") {
           text = $(
-            "You are at low risk of having COVID-19. You will still need to complete this risk " +
-            "assessment daily to monitor your symptoms."
+            "Complete this HealthCheck again in 7 days or sooner if you feel ill or you come into contact with someone infected with COVID-19."
           );
         }
         if (risk === "moderate") {
           text = $(
-            "You should SELF-QUARANTINE for 14 days and do HealthCheck daily to monitor " +
-            "symptoms. Try stay and sleep alone in a room that has a window with good air flow."
+            "We recommend you SELF-QUARANTINE for the next 14 days and do this HealthCheck daily to monitor your symptoms. Stay/sleep alone in a room with good air flow."
           );
         }
         if (risk === "high") {
           text = $([
-            "GET TESTED to find out if you have COVID-19.  Go to a testing center or Call " +
-            "0800029999 or your healthcare practitioner for info on what to do & how to test"
+            "You may be ELIGIBLE FOR COVID-19 TESTING. Go to a testing center or Call 0800029999 or visit your healthcare practitioner for info on what to do & how to test."
```

Comment:
Content in the translation sheet is
```
GET TESTED to find out if you have COVID-19. Go to a testing center or Call 0800029999 or your healthcare practitioner for info on what to do & how to test
```
So not sure why it's being changed.

Reply:
@rudigiesler which sheet are you getting all this information?

https://docs.google.com/spreadsheets/d/1cVjPb-ae1qOh4wxJ2skm0kS4QrUoCp9opm9Pqd59VGc/edit#gid=747997414

Reply:
@moh-moola That sheet is for the voluntary healthcheck. For the higher health healthcheck, it's this sheet https://docs.google.com/spreadsheets/d/1cVjPb-ae1qOh4wxJ2skm0kS4QrUoCp9opm9Pqd59VGc/edit#gid=859399454